src/containers/UserMenu.jsx

```diff
@@ -116,15 +116,18 @@ class UserMenu extends Component {
               {currentUser.email}
             </MenuItem>
             <Divider />
-            <Subheader>Teams</Subheader>
-            {currentUser.organizations.map(organization => (
-              <MenuItem
-                key={organization.id}
-                primaryText={organization.name}
-                value={organization.id}
-              />
-            ))}
-            <Divider />
+            {currentUser.organizations.length > 1 && (
+              <Subheader>Teams</Subheader>
+            )}
+            {currentUser.organizations.length > 1 &&
```

Comment:
Can we use the logic `currentUser.organizations.length > 1` only on the line 119 and put all the other code inside it? I think this way we keep the code more concise.

Reply:
@tom-raley pinging ya in here!